src/Http/Client.php

```diff
@@ -128,7 +126,7 @@ public function request(
 
         try {
             // Make a request with given parameters.
-            return new Response($this->client->request($method, $url, $options));
+            return new Response($this->httpClient, $method, $url, $options);
```

Comment:
@apotek Just checking if `->request()` was accidentally left off. I see the original has it.

Reply:
> @apotek Just checking if `->request()` was accidentally left off. I see the original has it.

You are right to ask about this. This change set changes the constructor of the Response class to no longer accept a GuzzleResponse (which is what client->request() would return), but instead accepts the request client object, and the request parameters:

https://github.com/ChromaticHQ/orange-dam-php/pull/40/files#diff-f6919b0f7fddf8ecc9da906c1a52a42e724966f3965be170e0ad30de2f2ee786L29-R37

This allows the constructor function in the Response object to save some data to itself about the request that is about to be made, so that the Response can be queried for data about the request that created that response.